src/main/java/org/locationtech/proj4j/proj/GeostationarySatelliteProjection.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.locationtech.proj4j.proj;
+
+import org.locationtech.proj4j.ProjCoordinate;
+import org.locationtech.proj4j.ProjectionException;
+import org.locationtech.proj4j.util.ProjectionMath;
+
+/**
+ *
+ * @author yaqiang
+ */
+public class GeostationarySatelliteProjection extends Projection {

```

Comment:
Do you want to add tests for this Projection? 
I understand that it can be hard to do, but at least some basic points transformations to track down possible regressions.

Reply:
The GeostationarySatelliteProjection.java code come from MeteoInfo project (https://github.com/meteoinfo/MeteoInfo/blob/master/MeteoInfoLib/src/main/java/org/meteoinfo/projection/proj4j/proj/GeostationarySatelliteProjection.java). It has been tested in real work of read and plot geostationary satellite data. An example image is attached.

![geos_proj](https://user-images.githubusercontent.com/4408029/54419448-6afeae00-4742-11e9-9e7a-ffe07afea093.PNG)